press/press/doctype/bench_update/bench_update.py

```diff
@@ -36,11 +36,12 @@ def deploy(self, apps_to_ignore):
 			apps_to_ignore = json.loads(self.apps_to_ignore)
 
 		rg: ReleaseGroup = frappe.get_doc("Release Group", self.group)
-		candidate = rg.create_deploy_candidate(apps_to_ignore)
-		candidate.deploy_to_production()
+		candidates = rg.create_deploy_candidate(apps_to_ignore)
+		for candidate in candidates:
+			candidate.deploy_to_production()
 
 		self.status = "Running"
-		self.candidate = candidate.name
+		self.candidate = candidates[0].name
```

Comment:
This will be very confusing when there can be more than 1 candidate but it will always link to first one.

Bad data structure to represent this IMO.